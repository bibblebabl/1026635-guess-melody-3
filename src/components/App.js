import React from 'react';
import WelcomeScreen from './WelcomeScreen';

const App = () => {
  return (
    <WelcomeScreen errors="Какая-та ошибка" />
  );
};

export default App;
