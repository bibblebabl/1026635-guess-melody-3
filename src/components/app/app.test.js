import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

it(`<App /> renders correctly`, () => {
  const AppComponent = renderer.create(<App />).toJSON();

  expect(AppComponent).toMatchSnapshot();
});
