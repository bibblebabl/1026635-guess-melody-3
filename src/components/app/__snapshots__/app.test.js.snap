// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="welcome"
>
  Какая-та ошибка
  <button
    className="button welcome__button"
    onClick={[Function]}
  >
    Start Game
  </button>
</div>
`;
